lib/streamlit/elements/widgets/select_slider.py

```diff
@@ -159,7 +159,7 @@ def select_slider(
             For accessibility reasons, you should never set an empty label (label="")
             but hide it with label_visibility if needed. In the future, we may disallow
             empty labels by raising an exception.
-        options : Sequence, numpy.ndarray, pandas.Series, pandas.DataFrame, or pandas.Index
+        options : Iterable
```

Comment:
I wish there was a way to indicate to users that `numpy.ndarray, pandas.Series, pandas.DataFrame, or pandas.Index`, etc are supported. Does it make sense to call it out in the `options` description? I can see how users make not make the connection that those types are Iterable..

Something to the effect of:

```
options : Iterable
            Labels for the select options. This will be cast to str internally
            by default. For pandas.DataFrame, the first column is used.
            Some examples of iterables include numpy.ndarray, pandas.Series,
            pandas.DataFrame, and pandas.Index.
```

?

Reply:
I've pushed changes.

Reply:
Thank you! We can merge now 😄